### PR TITLE
Popup menu v2

### PR DESCRIPTION
**Pull request by JakariaJishan**
- Adjust-font-size-stylesheet
- Change the border radious value of popup button

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -70,10 +70,10 @@ label {
   font-family: 'Inter', sans-serif;
```
Comment (christianbiring1) [inline style.css:70]: - [ ] Kindly make sure your pop-up modal is responsive both on desktop and mobile versions. As you may notice on your mobile design the user is not able to close the pop-up window. Kindly fix that.

Your mobile design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @JakariaJishan  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] The project section is rendered dynamically :+1: 
- [ ] The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project.

## Required Changes ♻️
- [x] `Kindly make sure your modal is responsive on all screen size`
- [x] `Kindly consider documenting your work in a professional way. You may consider providing as many details as possible in your README file to help people with no engineering background to use your repo`
`Hint: You may consider giving your project a descriptive name describing what it aims to do.`
`In the getting started section, Kindly you may consider proving as many details as possible like writing down the command and each step the user of the repo might use to get the full working project in their local environment.`
`e.g: clone <repo_link>`
`cd <dir_name>`
`...`
- [ ] `Since you did not run any kind of test in this project you may consider deleting that section.`
- [ ] `You may consider providing some info about how you deploy your page, on GitHub-pages`
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 > This is a simple replica of the template (https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27&t=9GufdfER8lvr1MpO-0) for portfolio setup.
 
-Live Link 👉  [portfolio](https://jakariajishan.github.io/portfolio-microverse/)
+Live Link 👉 [portfolio](https://jakariajishan.github.io/portfolio-microverse/)
 
 project presentation 👉 [presentation video](https://www.loom.com/share/0cb6ef04dbd54013b13744e93993467f)
```
Comment (algerina) [inline README.md]: - [ ] As requested by the previous reviewer kindly delete the Test section,

**Review by algerina — CHANGES_REQUESTED** [review]
Hi @JakariaJishan  
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## REQUIRED CHANGES 
kindly check the comments under the review.



 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Review by Olamarx — APPROVED** [review]
Hi @JakariaJishan,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed 👍🏼
- Commit messages ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**